gui/src/components/preboarding/admin-user.vue

```diff
@@ -320,6 +322,8 @@ export default class CortxAdminUser extends Vue {
 
   public async mounted() {
     await this.getSyetmStatus();
+    const getUnsupportedFeatures = this.$store.dispatch("userLogin/getUnsupportedFeaturesAction");

```

Comment:
Are these lines of code required(line no 325 and 326)? If not please remove them.

Reply:
Removed this code and updated router file to solve the purpose of this code.